packages/api/src/controllers/access-control.ts

```diff
@@ -135,10 +133,13 @@ app.post(
 
     const playbackPolicyType = content.playbackPolicy?.type ?? "public";
 
-    if (user.createdAt < HACKER_DISABLE_CUTOFF_DATE) {
```

Comment:
Wow I was re-reviewing this and just realized that this condition was inverted? Were we not limiting any new users (i.e. pirates) as well?? hahaha